### DISCUSSION TITLE
Update to utilize Depth-Anything v2 models?

### DISCUSSION THREAD

**Discussion by gituser123456789000** [BrokenSource/DepthFlow#32] [General]
Please update to use Depth-Anything v2 models if it doesn't already. I just found your project and it looks interesting, but I also see that it says it uses Depth-Anything, which I assume means the original v1. Just suggesting to add v2 support for potentially better results. Thanks.

**Comment by Tremeschin** [comment]
hey! thanks for getting in touch and wanting the best quality :)
 
it is your lucky day, v0.4.0 defaults to [depth anything v2](https://github.com/BrokenSource/DepthFlow/blob/86b5f9293ac3b728d7d3a29442e5dcb02a825e0c/DepthFlow/Scene.py#L65) and is available on [PyPI](https://brokensrc.dev/get/pypi) or running directly [from source](https://brokensrc.dev/get/source) (no binary releases yet)

there's some example commands when you just run `depthflow` on this latest version for changing the estimator, using an image upscaler, and basic settings configuration (which I'll document on the website soon-ish)

can't deny v2 is better and more accurate than v1, but overall, the final video looks practically the same with fewer relative depth innacuracies

**Reply by gituser123456789000** [reply]
> hey! thanks for getting in touch and wanting the best quality :)
> 
> it is your lucky day, v0.4.0 defaults to [depth anything v2](https://github.com/BrokenSource/DepthFlow/blob/86b5f9293ac3b728d7d3a29442e5dcb02a825e0c/DepthFlow/Scene.py#L65) and is available on [PyPI](https://brokensrc.dev/get/pypi) or running directly [from source](https://brokensrc.dev/get/source) (no binary releases yet)
> 
> there's some example commands when you just run `depthflow` on this latest version for changing the estimator, using an image upscaler, and basic settings configuration (which I'll document on the website soon-ish)
> 
> can't deny v2 is better and more accurate than v1, but overall, the final video looks practically the same with fewer relative depth innacuracies

so v0.4.0 is coming soon? or am I missing it and it's out already? I see 0.3.1

**Reply by gituser123456789000** [reply]
I'm also not understanding the install. I got 0.3.1, run the exe, there's a popup box that says downloading the distribution followed by unpacking the distribution.. then File Explorer has a loading indicator on the mouse for a while.. and that's it. I see no folder created, no application to run..

**Reply by Tremeschin** [reply]
I've recently split all projects into individual packages, but due to a lack of a feature I need in PyApp to do it in a certain way, I can't create executable releases until I revisit this problem

there's multiple places discouraging them as they are experimental in general, but yea, it's technically already out on where I've linked in the previous message (on the repo or Python's package registry)

about "no window to run", see [this orange warning](https://brokensrc.dev/get/releases/#usage)* (tldr open it from a powershell, there's a command line interface that expects some stuff)

<sub>* (it's a bit outdated as calling `depthflow main` is explicit for a while now to see all available options, use upscalers, change depth estimator, etc, but for other running methods you'll already be on a terminal anyways) </sub>
